edenai_apis/apis/astria/astria_api.py

```diff
@@ -26,30 +31,81 @@ def __init__(self, api_keys: Dict = {}) -> None:
         self.headers = {"authorization": f"Bearer {self.api_key}"}
 
     def image__generation_fine_tuning__create_project_async__launch_job(
-        self,
-        name: str,
-        description: str,
-        files: List[str],
-        files_url: List[str] = [],
-        base_project_id: Optional[int] = None,
+            self,
+            title: str,
+            class_name: str,
+            files: List[str] = [],
+            files_url: List[str] = [],
+            base_tune_id: Optional[int] = None,
     ) -> AsyncLaunchJobResponseType:
-        raise NotImplementedError
+        data = {
+            "tune[title]": title,
+            "tune[name]": class_name,
+            "tune[base_tune_id]": base_tune_id,
+            # "tune[callback]": 'https://optional-callback-url.com/to-your-service-when-ready?prompt_id=1'
+        }
+        for image in files:
+            image_data = load_image(image)  # Assuming image is a file path
+            files.append(("tune[images][]", image_data))
+        for image_url in files_url:
+            files.append(("tune[image_urls][]", image_url))
+
+        response = requests.post(f"{self.url}tunes", data=data, files=files, headers=self.headers)
+        response.raise_for_status()
+        return AsyncLaunchJobResponseType(provider_job_id=response.json()["id"])
```

Comment:
provider_job_id is a string not an integer

Reply:
fixed